src/__tests__/lib/containers/QueryWrapperMenu.test.tsx

```diff
@@ -170,4 +205,77 @@ describe('it renders an accordion config', () => {
     expect(wrapper.find(ACCORDION_GROUP_MENU_SELECTOR).childAt(2).hasClass('SRC-pointed-triangle-right')).toBeTruthy()
   })
 
+  it('passes down the correct unitDescription correctly' , async () => {
+    const { wrapper } = await createMountedComponent(props)
+    const queryWrapper = wrapper.find(QueryWrapper).at(0)
+    expect(queryWrapper.props().unitDescription).toEqual('Computational Tools by a')
+  })
+
 })
+
+describe('it renders with search correctly configured ', () => {
+  const SynapseClient = require('../../../lib/utils/SynapseClient')
+  SynapseClient.getQueryTableResults = jest.fn(() => Promise.resolve(mockData))
+  const sql: string = 'SELECT * FROM syn5604373'
+  const genericCardSchema: GenericCardSchema = {
+    type: 'PROGRAM',
+    title: 'title',
+    subTitle: 'subtitle',
+    description: 'description',
+    icon: 'icon',
+    link: 'link',
+    secondaryLabels: {
+      0: { key: 'labelOne', alias: 'Label One' },
+      1: { key: 'labelTwo' },
+    }
+  }
+  const props: QueryWrapperMenuProps = {
+    accordionConfig: [
+      {
+        name: 'Computational',
+        menuConfig: [{ sql, facet: 'a' }, { sql, facet: 'b' }, { sql, facet: 'c' }],
+        cardConfiguration: {
+          type: SynapseConstants.GENERIC_CARD,
+          genericCardSchema
+        },
+        searchConfiguration: {
+          searchable: [
+            {
+              columnName: 'a',
+              hintText: 'hint'
+            }
+          ]
+        }
+      },
+      {
+        name: 'Experimental',
+        menuConfig: [{ sql, facet: 'd' }, { sql, facet: 'e' }],
+        cardConfiguration: {
+          type: SynapseConstants.GENERIC_CARD,
+          genericCardSchema
+        },
+        searchConfiguration: {
+          searchable: [
+            {
+              columnName: 'd',
+              hintText: 'hint'
+            }
+          ]
+        }
+      } 
+    ],
+    rgbIndex: 3,
+  }
+
+  it('renders correctly and passes down the right unitDescription', async () => {
+    const { wrapper } = await createMountedComponent(props)
+    const queryWrapperWithSearch = wrapper.find(QueryWrapper).at(2)
+    expect(queryWrapperWithSearch.props().unitDescription).toEqual('Computational Tools')
+    // Search only makes sense with an unfaceted view, so there's additional logic to make sure only
+    // in the case of search that facets are excluded in QueryWrapperMenu
+    expect(queryWrapperWithSearch.find(Facets)).toHaveLength(0)
```

Comment:
this is the only configuration when facets aren't shown, we also have to test unitDescription is shown correctly